include/cpp2util.h

```diff
@@ -945,7 +946,8 @@ inline auto to_string(char const& t) -> std::string
     return std::string{t};
 }
 
-inline auto to_string(char const* s) -> std::string
+template <std::same_as<char const*> X>
+inline auto to_string(X& s) -> std::string
```

Comment:
```suggestion
inline auto to_string(const X& s) -> std::string
```
Otherwise, can't pass a `const` `char const*`: <https://cpp2.godbolt.org/z/q44111ezM>.

Reply:
fixed